conbench/tests/app/test_users.py

```diff
@@ -142,6 +143,7 @@ def test_user_create_post_unauthenticated(self, client):
 
     def test_user_create_post_no_csrf_token(self, client):
         self.authenticate(client)
-        response = client.post("/users/create/", data={})
+        data = {"name": "New Name", "email": "test@test.com"}
```

Comment:
I had to fix this because some kind of package upgrade causes this too fail with 400 now.

Reply:
Thanks for these!